src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -504,21 +505,27 @@ protected function initializeContainer()
                     return;
                 }
 
-                $backtrace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS, 3);
+                $backtrace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS, 5);
                 // Clean the trace by removing first frames added by the error handler itself.
                 for ($i = 0; isset($backtrace[$i]); ++$i) {
                     if (isset($backtrace[$i]['file'], $backtrace[$i]['line']) && $backtrace[$i]['line'] === $line && $backtrace[$i]['file'] === $file) {
                         $backtrace = \array_slice($backtrace, 1 + $i);
                         break;
                     }
                 }
+                for ($i = \count($backtrace) - 2; 0 < $i; --$i) {
```

Comment:
I would add the explanation comment before this loop, as done for the previous one.

Reply:
comment added